pkg/api/validation/validation.go

```diff
@@ -2045,3 +2052,21 @@ func ValidateLoadBalancerStatus(status *api.LoadBalancerStatus) errs.ValidationE
 	}
 	return allErrs
 }
+
+// ValidateEvent makes sure that the event makes sense.
+func ValidateEvent(event *api.Event) errs.ValidationErrorList {
+	allErrs := errs.ValidationErrorList{}
+
+	requiresNamespace := true
+	if event.InvolvedObject.Kind == "Node" {
```

Comment:
I see it's not worse than it was, so not asking for changes unless you feel like adding a TODO.